### DISCUSSION TITLE
Course-央美×中传游研社 电子游戏史 | Zi-Zi's Journey

### DISCUSSION THREAD

**Discussion by giscus** [GZ-Metal-Cell/GZ-Metal-Cell.github.io#345] [Announcements]
# Course-央美×中传游研社 电子游戏史 | Zi-Zi's Journey

电子游戏史。

https://gz-metal-cell.github.io/2024/09/30/Course-%E5%A4%AE%E7%BE%8E%C3%97%E4%B8%AD%E4%BC%A0%E6%B8%B8%E7%A0%94%E7%A4%BE%20%E7%94%B5%E5%AD%90%E6%B8%B8%E6%88%8F%E5%8F%B2/

**Comment by promefire** [comment]
VR游戏何在!

**Reply by GZ-Metal-Cell** [reply]
个人认为，VR 游戏目前还不是很流行（我甚至从来没有体验过）。不过，我听说著名游戏开发专家伪文青 promefire 将于 2025 年开发一款 VR 剧情向游戏《Promefire's Journey》，这款具有划时代意义的巨作值得期待！在不久的将来就可以作为著名案例被引用到课堂当中了！

**Reply by promefire** [reply]
现在v我50，以后上教材带你名字😘

**Reply by GZ-Metal-Cell** [reply]
带上我这个菜狗后就变反面教材了😭
